apps/condo/domains/user/schema/ConfirmPhoneAction.js

```diff
@@ -102,14 +135,51 @@ const ConfirmPhoneAction = new GQLListSchema('ConfirmPhoneAction', {
     },
 })
 
-
 const ConfirmPhoneActionService = new GQLCustomSchema('ConfirmPhoneActionService', {
-    queries: [
+    types: [ 
+        {
+            access: true,
+            type: 'input GetPhoneByConfirmPhoneActionTokenInput { token: String!, captcha: String! }',
+        },
+        {
+            access: true,
+            type: 'type GetPhoneByConfirmPhoneActionTokenOutput { phone: String! }',
+        },
         { 
             access: true,
-            schema: 'getPhoneByConfirmPhoneActionToken(token: String!): String',
+            type: 'input StartConfirmPhoneActionInput { phone: String!, dv:Int!, sender: JSON!, captcha: String! }',
+        },
+        {
+            access: true,
+            type: 'type StartConfirmPhoneActionOutput { token: String! }',
+        },    
+        { 
+            access: true,
+            type: 'input ConfirmPhoneActionResendSmsInput { token: String!, sender: JSON!, captcha: String! }',
+        },
+        {
+            access: true,
+            type: 'type ConfirmPhoneActionResendSmsOutput { status: String! }',
+        },   
+        { 
+            access: true,
+            type: 'input CompleteConfirmPhoneActionInput { token: String!, smsCode: Int!, captcha: String! }',
+        },
+        {
+            access: true,
+            type: 'type CompleteConfirmPhoneActionOutput { status: String! }',
+        },                   
+    ],    
+    queries: [ 
+        { 
+            access: true,
```

Comment:
It looks like this mutation is used only for tests. It's better to close access only for admin.

Reply:
This query is used for the user to get information about the phone he is trying to confirm. As registration uses 3 different screens and we have only token in URL we need to get phone number from token in case the page reload